CHANGES.md

```diff
@@ -48,6 +48,7 @@ Features
 * [#621](https://github.com/java-native-access/jna/pull/621): Added TYPEFLAGS-constants for `wTypeFlags` in `com.sun.jna.platform.win32.OaIdl.TYPEATTR` - [@SevenOf9Sleeper](https://github.com/SevenOf9Sleeper).
 * [#625](https://github.com/java-native-access/jna/pull/625): Make conversion to/from java to/from VARIANT in `com.sun.jna.platform.win32.COM.util.Convert` more flexible and dependable - [@matthiasblaesing](https://github.com/matthiasblaesing).
 * [#639](https://github.com/java-native-access/jna/pull/639): Add getloadavg() to OS X and Unix - [@dbwiddis](https://github.com/dbwiddis).
+* [#640](https://github.com/java-native-access/jna/pull/640): Add GetPerformanceInfo() to Win32.Psapi - [@dbwiddis](https://github.com/dbwiddis).
```

Comment:
This added GetTickCount64 too, can you please amend? And make the whole namespace here like above, so `com.sun...`, cause not everyone is on Windows :)